### PR TITLE
CHANGE: Remove Tanks demo.

### PR DESCRIPTION
**Pull request by Rene-Damm**
Nuke `TanksDemo` and link to [Warriors demo](https://github.com/UnityTechnologies/InputSystem_Warriors) in docs.

### CODE REVIEW

**Review by karljj1 — APPROVED** [review]

File: Packages/com.unity.inputsystem/Documentation~/Installation.md [hunk 1]
```diff
@@ -35,3 +35,5 @@ When the new input backends are enabled, the `ENABLE_INPUT_SYSTEM=1` C# `#define
 The Input System package comes with a number of samples. You can install these directly from the Package Manager window in Unity (menu: __Window > Package Manager__). To see the list, select the Input System package in the Package Manager window. Click **Import into Project** next to a sample to copy it into the current Project.
 
 ![Install Samples](Images/InstallSamples.png)
+
+A more comprehensive demo project for the Input System can be found on GitHub under [InputSystem_Warriors](https://github.com/UnityTechnologies/InputSystem_Warriors).
```
Comment (keiramoon) [inline Packages/com.unity.inputsystem/Documentation~/Installation.md]:
```suggestion
For a more comprehensive demo project for the Input System, see the [InputSystem_Warriors](https://github.com/UnityTechnologies/InputSystem_Warriors) GitHub repository.
```

File: Packages/com.unity.inputsystem/CHANGELOG.md [hunk 2]
```diff
@@ -12,6 +12,8 @@ however, it has to be formatted properly to pass verification tests.
 ### Changed
 
 - The `submit` and the `cancel` actions of the UI input module now trigger on __release__ instead of press. This makes the behavior consistent with clicks triggering UI response on release rather than press.
+- We removed the old "Tanks" demo (previously available from the samples shipped with the package).
```
Comment (keiramoon) [inline Packages/com.unity.inputsystem/CHANGELOG.md]:
```suggestion
- Removed the old "Tanks" demo (previously available from the samples shipped with the package).
```

(per Style Guide, we don't say "we").

File: Packages/com.unity.inputsystem/CHANGELOG.md [hunk 3]
```diff
@@ -12,6 +12,8 @@ however, it has to be formatted properly to pass verification tests.
...
+  * We now have a much better demo in the form of the [Warriors demo](https://github.com/UnityTechnologies/InputSystem_Warriors).
```
Comment (keiramoon) [inline Packages/com.unity.inputsystem/CHANGELOG.md]:
```suggestion
  * Added a new and improved demo project, which you can download from the [InputSystem_Warriors](https://github.com/UnityTechnologies/InputSystem_Warriors) GitHub repository.
```

**Review by keiramoon — APPROVED** [review]
Approved, but see suggestions :)

File: Packages/com.unity.inputsystem/CHANGELOG.md (hunk 3 above)
Comment (Rene-Damm) [reply Packages/com.unity.inputsystem/CHANGELOG.md]: Pushed

File: Packages/com.unity.inputsystem/CHANGELOG.md (hunk 2 above)
Comment (Rene-Damm) [reply Packages/com.unity.inputsystem/CHANGELOG.md]: Pushed

File: Packages/com.unity.inputsystem/Documentation~/Installation.md (hunk 1 above)
Comment (Rene-Damm) [reply Packages/com.unity.inputsystem/Documentation~/Installation.md]: Pushed

**Pull request merged**